src/main/java/world/bentobox/bentobox/api/panels/Panel.java

```diff
@@ -190,18 +186,15 @@ public void setUser(User user) {
     @Override
     public void setHead(PanelItem item) {
         // Update the panel item
-        // Replace the item in the item list if the name is the same
-        items = items.entrySet().stream()
-                .collect(Collectors.toMap(Map.Entry::getKey, e -> (item.getName().equals(e.getValue().getName()) ? item : e.getValue())));
-        // Replace the inventory slot item
-        for (int i = 0; i < inventory.getSize(); i++) {
-            ItemStack it = inventory.getItem(i);
-            if (it != null && it.getType().equals(Material.PLAYER_HEAD)) {
-                ItemMeta meta = it.getItemMeta();
-                if (meta != null && ChatColor.stripColor(item.getName()).equals(ChatColor.stripColor(meta.getLocalizedName()))) {
-                    inventory.setItem(i, item.getItem());
-                }
-            }
+        // Find panel item index in items and replace it once more in inventory to update it.
+
+        OptionalInt index = this.items.entrySet().stream().
+            filter(entry -> entry.getValue() == item).
+            mapToInt(Map.Entry::getKey).findFirst();
```

Comment:
This looks like it will only change one head in an panel. Will it? In the Top Ten for example, the player head can be shown twice in the panel. Once in the top ten, once at the bottom line. Please confirm this use case is handled.